src/kinetics/PlogRate.cpp

```diff
@@ -120,6 +116,14 @@ void PlogRate::setRates(const std::multimap<double, ArrheniusRate>& rates)
         j++;
         rates_.push_back(rate.second);
     }
+    if (!m_valid) {
+        // ensure that reaction rate can be evaluated (but returns NaN)
+        rates_.reserve(2);
```

Comment:
Is this needed before `push_back()` is used?

Reply:
Not needed, but good practice.